tests/test-ffarith.C

```diff
@@ -420,6 +428,9 @@ int main(int argc, char ** argv)
     TEST_SPECIFIC(Extension<GFqDom<int64_t> >, GF11E9, GF11E3, 3);
     TEST_SPECIFIC(Extension<>, GF13E8, 13, 8);
 
+        // Rational numbers
+
+    TEST_SPECIFIC(QField<Rational>, QQ, 0);
```

Comment:
Mmm, is QQ a finite field?
Maybe rename the file into test-fieldarith.C

Reply:
done
in commit 8496937afebd4ff54ed1696e2983324dbffb147b